dashboard/test/models/level_test.rb

```diff
@@ -212,6 +212,15 @@ def raise_unless_specifies_ideal_level_source(level_class)
       level.properties['initial_dirt']
   end
 
+  test 'creating custom level from file sets level_concept_difficulty' do
+    Level.find_by_name('K-1 Bee 2')&.destroy
```

Comment:
Note, we already have a dependency on `K-1 Bee 2.level` in this test file (see the test above this one) so I stuck with that for now; I'm setting up proper level file fixtures in my seeding tests on the other PR though.